app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -43,8 +43,15 @@ menuentry 'Chainload Grub2 EFI from ESP' --id local_chain_hd0 {
 -%>
   echo Partition with known EFI file not found, you may want to drop to grub shell
   echo and investigate available files updating 'pxegrub2_chainload' template and
-  echo the list of known filepaths for probing. Contents of /EFI directory:
-  ls ($chroot)/EFI
+  echo the list of known filepaths for probing. Available devices are:
+  echo
+  ls
+  echo
+  If you can't see the HDD, make sure the drive is marked as bootable in EFI and
+  not hidden. Boot order must be the following:
+  1) NETWORK
+  2) HDD
```

Comment:
I'm not very familiar with pxegrub syntax, but shouldn't these be?
```suggestion
  echo If you can't see the HDD, make sure the drive is marked as bootable in EFI and
  echo not hidden. Boot order must be the following:
  echo 1) NETWORK
  echo 2) HDD
```

Reply:
Forgot to repeat `:normal Iecho ` in Vim on the other lines :)